testing/fuchsia/run_tests.sh

```diff
@@ -68,3 +68,8 @@ done
     -f shell_tests-0.far  \
     -t shell_tests \
     -a "--gtest_filter=-ShellTest.HandlesActualIphoneXsInputEvents:ShellTest.CacheSkSLWorks:ShellTest.SetResourceCacheSize*:ShellTest.Screenshot:ShellTest.WaitForFirstFrameTimeout"
+
+echo "Dumping system logs..."
+
+./fuchsia_ctl -d $device_name ssh \
+    -c "log_listener --dump_logs yes"
```

Comment:
very nice! But this should move into the reboot trap so that we do it if a test fails.  I'm pretty sure this won't happen if a test fails today.